stable/restore/README.md

```diff
@@ -150,11 +150,14 @@ The following tables list the configurable parameters of the restore chart and t
 
 | Parameter | Description | Default |
 | ----- | ----------- | ------ |
-| `restore.type` | What type of restore to perform: [ "database" | "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
-| `restore.target` | Where to restore `TO` | `{{ .Values.database.name }}` |
-| `restore.source` | Where to restore `FROM` [ backupset | url | `:latest` ] | `:latest` |
+| `restore.type` | What type of restore to perform: [ "database" \| "archive" ]. A "database" restore restarts the entire database at a previous state. An "archive" restore restores/repairs a SINGLE archive in a RUNNING database. | `"database"` |
+| `restore.target` | Where to restore `TO` | `"demo"` |
+| `restore.source` | Where to restore `FROM` [ _backupset_ \| _url_ \| `:latest` \| `:group-latest` ] | `:latest` |
```

Comment:
What do `TO` and `FROM` mean here, and why are they formatting this way (normally fixed width is used to denote code, but think it is just used for emphasis here)? The description doesn't actually say what these values should be, though I can guess that `restore.target` is a database name and `restore.source` is a backup set or URL. I'm not sure how we would differentiate between the two; perhaps the protocol/scheme is used to determine that it is a URL, otherwise it is interpreted as a backup set. We should document that.

Reply:
Good point. I'll edit the description as suggested.